### ISSUE TITLE
Combining FASTQ files

### ISSUE THREAD

**Issue by denisemauldin** [NickSto/duplex#2]
Hi there,

When I run this step:

```
cat reads_1.fastq | paste - - - -   | paste - <(cat reads_2.fastq | paste - - - -) 
```

It gives me a file that is:
Read1_name  Read1_sequence Read1_+line Read1_quality Read2_name Read2_sequence Read2_+line Read2_quality

That doesn't match up with the expected input of make-barcodes.awk, which (from header) is:
    # Input columns (the 4 FASTQ lines for both reads in a read pair):
    #   1: read1 name
    #   2: read2 name
    #   3: read1 sequence
    #   4: read2 sequence
    #   5: read1 + line
    #   6: read2 + line
    #   7: read1 quality
    #   8: read2 quality

Is there an updated command?  Should I just update the make-barcodes.awk?

Thanks,
Denise

**Comment by NickSto** [comment]
Hi Denise,

I'm sorry I missed your report! My notifications have been pretty non-functional until now. This repository is no longer the canonical one, and it's far behind the official one at [galaxyproject/dunovo](https://github.com/galaxyproject/dunovo). I've added a note warning about this, but in the future I may just delete this repository.

In the current version, I've corrected the documentation to match the new version of `make-barcodes.awk`. The new command is

``` bash
paste reads_1.fastq reads_2.fastq | paste - - - -
```

which removes the unusual `<(command)` syntax which is not supported in some shells.
